eventstore/mongodb/eventstore.go

```diff
@@ -82,7 +82,7 @@ func NewEventStoreWithSession(session *mgo.Session, database string) (*EventStor
 }
 
 type aggregateRecord struct {
-	AggregateID string         `bson:"_id"`
+	AggregateID interface{}    `bson:"_id"`
```

Comment:
If I store `AggregateID` as string (or other type), mongo will not be able to unmarshall this type to `ID` directly, and we will need a convertion function between types, like `SetBSON`